source/stylesheets/refills/_animate.scss

```diff
@@ -0,0 +1,87 @@
+$base-accent-color: #477dca !default;
+
+.animated {
+  -webkit-animation-duration: 1s;
```

Comment:
I wanted to keep this as true as possible to the original `animate.css` code. I'm emailing Dan who built them to see if we can do this before going ahead.